server/src/client_service.rs

```diff
@@ -455,6 +470,88 @@ mod tests {
         assert_eq!(quotes2, quotes);
     }
 
+    #[test_with_logger]
+    fn submit_quotes_rejects_quotes_above_max_simultaneous_quotes(logger: Logger) {
+        let pair = Pair {
+            base_token_id: TokenId::from(1),
+            counter_token_id: TokenId::from(2),
+        };
+        let mut rng: StdRng = SeedableRng::from_seed([1u8; 32]);
+        let quote_book = InMemoryQuoteBook::default();
+        let (client_api, _server, _msg_bus_tx, _msg_bus_rx) =
+            create_test_client_and_server(&quote_book, &logger);
+
+        // Submitting SCIs under MAX_Simultaneous_Quote should be successful
```

Comment:
```suggestion
        // Submitting SCIs under MAX_SIMULTANEOUS_QUOTES should be successful
```

Reply:
Yeah, I'm not sure why that happened. I think maybe I force pushed over it? Sorry about that.